### PR TITLE
Utilising Util Methods

### PR DESCRIPTION
**Pull request by Spedwards**

### CODE REVIEW

**Comment by Spedwards** [comment]
Up to now, I've gone through a few files, and I'm currently just replacing common expressions with Util methods.

Once this merges, I will start replacing the calls to functions in `global.js` to calls to the methods in `util.js`.

**Comment by chobobobo** [comment]
This needs to wait for visualsRefactor to be merged first.

File: room.js [hunk 1]
```diff
@@ -15,79 +15,77 @@ mod.extend = function(){
...
+                        const byType = c => c.controller === true;
+                        let r = _.filter(this.all, byType);
                         // add managed
-                        let isEmpty = c => c.sum <= (c.storeCapacity * MANAGED_CONTAINER_TRIGGER);
-                        this._out = this._out.concat(this.managed.filter(isEmpty));
-                    }
-                    return this._out;
+                        const isEmpty = c => c.sum <= (c.storeCapacity * MANAGED_CONTAINER_TRIGGER);
+                        r = r.concat(this.managed.filter(isEmpty));
+                    });
```
Comment (chobobobo) [inline room.js]: missing return

Comment (Spedwards) [inline room.js]: Good catch!

**Comment by Spedwards** [comment]
I'm going to leave this for separate file refactors. It's too big of a change with too many moving parts.

**Pull request closed without merging**